src/modules/lighthouse/auditUrl.ts

```diff
@@ -0,0 +1,38 @@
+import ora from 'ora'
+import chalk from 'chalk'
+
+import log from '../../logger'
+import { lighthouse, workspaces } from '../../clients'
+import { getAccount, getWorkspace } from '../../conf'
+import { TableGenerator } from './TableGenerator'
+
+const [account, currentWorkspace] = [getAccount(), getWorkspace()]
+
+async function isProdutionWorkspace(): Promise<boolean> {
+  const meta = await workspaces.get(account, currentWorkspace)
+  return meta.production
+}
+
+export default async (url: string, option: any) => {
+  if (await isProdutionWorkspace()) {
+    log.error(`You can not run lighthoust audits on production workspaces.`)
```

Comment:
_typo_: lighthoust -> lighthouse?

Reply:
Maybe `You cannot` or `You can't`? Even though all are correct it seems more common to use `can't` or `cannot`